index.bs

```diff
@@ -4738,6 +4745,41 @@ RogueList when performing ECDAA-Verify (see section 3.6 in [[!FIDOEcdaaAlgorithm
 [[FIDOMetadataService]] provides one way to access such information.
 
 
+## Security Benefits for [RPS] ## {#sctn-rp-benefits}
+
+The main benefits offered to [=[RPS]=] by this specification include:
+
+1. Users and accounts can be secured using widely compatible, easy-to-use multi-factor authentication.
+1. The [=[RP]=] does not need to provision [=authenticator=] hardware to its users. Instead, each user can independently obtain
+    any conforming [=authenticator=] and use that same [=authenticator=] with any number of [=[RPS]=].
+1. [=Registration=] and [=authentication=] [=ceremonies=] are highly resistant to [=man-in-the-middle attacks=].
+1. The [=[RP]=] can automatically support multiple types of [=user verification=] - for example PIN, biometrics and/or future
+    methods - with little or no code change, and can let each user decide which they prefer to use via their choice of
+    [=authenticator=].
+1. The [=[RP]=] does not need to store any secrets or sensitive information in order to gain the above benefits.
+
+As stated in the [[#conforming-relying-parties|Conformance]] section, the [=[RP]=] MUST behave as described in [[#rp-operations]]
+to obtain all of the above security benefits. However, one notable use case that departs slightly from this is described in the
+next section.
+
+
+### Self Attestation and Ignoring Attestation ### {#sctn-no-attestation-security-attestation}
+
+When [[#registering-a-new-credential|registering a new credential]], the [=[RP]=] MAY choose to accept an [=attestation
+statement=] with [=self attestation=] or [=no attestation statement|no attestation=], or to not verify the [=attestation
+statement=] at all. In all of these cases the [=[RP]=] loses much of benefit (3) listed above, but retains the other benefits.
+
+In these cases it is possible for a [=man-in-the-middle attack|man-in-the-middle attacker=] - for example, a malicious [=client=]
+or script - to replace the [=credential public key=] to be registered, and subsequently tamper with any future [=assertion=]
```

Comment:
1. Fixed.
2. Yes, that's what I'm imagining.

I think for assertion, the attack code would only need to implement signing `authData || clientDataHash` (where `authData` can be a constant per RP) with the private key. For registration, if the RP accepts None attestation, it's enough to generate a fixed-length random credential ID and construct the attestation object as `prefix || credentialId || suffix`, where `prefix` is constant per RP and `suffix` is globally constant. For self attestation it would also be necessary to sign the `authData || clientDataHash` and return `prefix || credentialId || infix || signature || suffix`, where `infix` is also globally constant.